src/lib/containers/AddToDownloadListV2.tsx

```diff
@@ -41,6 +41,7 @@ const AddToDownloadListV2: React.FunctionComponent<
   return (
     <>
       <a
+        data-testid="AddToDownloadListV2"
```

Comment:
I don't like using data-testid over having accessible roles, but I think making SynapseTable truly accessible and having more complete test cases is its own unit of work